spec/metrician_spec.rb

```diff
@@ -10,12 +10,32 @@ def self.null_agent(agent_class: Instrumental::Agent)
 RSpec.describe Metrician do
   before(:each) do
     Metrician.reset
+    ENV["METRICIAN_CONFIG"] = nil
```

Comment:
I like this change because the intent is more clear, but something tickled my brain about it so I looked it up.

From https://ruby-doc.org/core-1.9.3/ENV.html regarding `delete`:
> Sets the environment variable name to value. If the value given is nil the environment variable is deleted.

So the functionality is actually the same!